### DISCUSSION TITLE
Stereo Tool on AzuraCast Liquidsoap

### DISCUSSION THREAD

**Discussion by sb-200** [savonet/liquidsoap#1925] [Q&A]
Hi guys, 

I'm wondering if anyone has had success using stereo tool with Liquidsoap on their AzuraCast installation?
I have been trying to get this to work for days but had no success. using Stereo Tool Command Line, I updated the docker compose file and inserted the pipe code to the liquid soap configuration to no avail. 

any help would be greatly appreciated as I know there are others trying to do this too :)

<img width="1066" alt="Screenshot 2021-09-14 at 15 27 29" src="https://user-images.githubusercontent.com/84730588/133276492-481a7cb7-3fd9-4def-9dbd-157c01ee68a2.png">

<img width="1066" alt="image0" src="https://user-images.githubusercontent.com/84730588/133276513-b8922e65-1ef6-4ff2-a01a-ad1ecda7a078.JPG">

**Comment by toots** [comment]
Hi!

What issues are you seeing?

**Reply by sb-200** [reply]
> Hi!
> 
> What issues are you seeing?

hi there, 

nothing obvious to me is standing out in the logs , it seems to be loading ok, but there is no audio. 

my apologies if I'm missing something simple, this is mostly new to me!

many thanks

<img width="784" alt="Screenshot 2021-09-14 at 16 11 16" src="https://user-images.githubusercontent.com/84730588/133284279-8b2ad9e6-7bb4-4672-83a4-2dc29e111d09.png">

**Reply by sb-200** [reply]
<img width="784" alt="Screenshot 2021-09-17 at 00 30 56" src="https://user-images.githubusercontent.com/84730588/133701577-818c20b7-fed9-445d-beb9-16c0e39bdc9e.png">
<img width="854" alt="Screenshot 2021-09-17 at 00 31 50" src="https://user-images.githubusercontent.com/84730588/133701599-73027ad5-22e9-4239-a3ec-dee527c6994a.png">

**Reply by sb-200** [reply]
<img width="752" alt="Screenshot 2021-09-17 at 00 34 17" src="https://user-images.githubusercontent.com/84730588/133701736-f5304b39-1056-489f-ba7d-38ab46c634d3.png">

**Reply by toots** [reply]
That looks like the beginning of something. I think my first approach would be to try a very minimal setup, maybe a simple script with a `sine` input, the pipe and a `output.icecast`..?
